TECHMANIA/Assets/Scripts/Components/Main Menu/SelectTrackPanel.cs

```diff
@@ -743,109 +743,108 @@ private void BuildStreamingTrackList()
         {
             return;
         }
-
+        
+        // Get all track.tech files.
         string[] relativeTrackFiles = BetterStreamingAssets.GetFiles(
             Paths.RelativePathInStreamingAssets(
-                Paths.GetStreamingTrackRootFolder()),
+                Paths.GetStreamingTrackRootFolder()
+            ),
             Paths.kTrackFilename,
-            SearchOption.AllDirectories);
+            SearchOption.AllDirectories
+        );
+
+        // Get all directories above them.
         foreach (string relativeTrackFile in relativeTrackFiles)
         {
-            string trackFile = Paths.AbsolutePathInStreamingAssets(
-                relativeTrackFile);
-            string trackFolder = Path.GetDirectoryName(trackFile);
-            string trackParent = Path.GetDirectoryName(trackFolder);
-            if (trackParent == Paths.GetStreamingTrackRootFolder())
```

Comment:
I notice this check is removed. It doesn't matter at the moment, but it does prevent us from placing tracks at the root of streaming track folder.